### PR TITLE
add oct18 road update post

### PR DESCRIPTION
**Pull request by gregbunce**
and update the ng911 database date

### CODE REVIEW

File: _posts/2018-10-11-utah-sgid-statewide-roads-data-layer-updates-10112018.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Note: This update includes two separate road updates, spanning September and October:
+
+#### County Updates
+New roads were added and road names and address ranges were updated for the following counties:
+
+- **Cache County:** Obtained roads data on 08/13/2018. Previous update was on 06/27/2018.
+
+- **Davis County:** Obtained roads data on 09/10/2018. Previous update was on 08/14/2018.
+
+- **Salt Lake County (2 updates):** Obtained roads data on 08/09/2018 and 09/10/2018. Previous update was on 08/14/2018.
```
Comment (stdavis) [inline _posts/2018-10-11-utah-sgid-statewide-roads-data-layer-updates-10112018.md]: These dates look a little off...

File: _posts/2018-10-11-utah-sgid-statewide-roads-data-layer-updates-10112018.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+- **San Juan County:** Obtained roads data on 08/16/2018. Previous update was on 01/2016.
+
+- **Summit Juan County:** Obtained roads data on 08/30/2018. Previous update was on 05/30/2018.
+
+- **Tooele Juan County:** Obtained roads data on 08/10/2018. Previous update was on 06/27/2018.
+
+- **Utah County (2 updates):** Obtained roads data on 08/09/2018 and 09/10/2018. Previous update was on 08/14/2018.
```
Comment (stdavis) [inline _posts/2018-10-11-utah-sgid-statewide-roads-data-layer-updates-10112018.md]: Same with these...

File: _posts/2018-10-11-utah-sgid-statewide-roads-data-layer-updates-10112018.md (hunk 1 above)
Comment (gregbunce) [reply _posts/2018-10-11-utah-sgid-statewide-roads-data-layer-updates-10112018.md]: Yeah, it's an odd one this month (two updates in one plus odd obtained dates).  It took me a bit, too, to wrap my head around them.  But, oddly, they are correct.  Thanks for checking this type of stuff.

File: _posts/2018-10-11-utah-sgid-statewide-roads-data-layer-updates-10112018.md (hunk 2 above)
Comment (gregbunce) [reply _posts/2018-10-11-utah-sgid-statewide-roads-data-layer-updates-10112018.md]: Again, odd dates.. but correct.  I double checked them.  Thanks again for the level of detail that you're checking this!  I appreciate it.

**Comment by gregbunce** [comment]
I added two last min commit to fix county names.  Just wanted to make sure that made it into the pull request.j thanks @stdavis

**Comment by stdavis** [comment]
OK. I think that I see what the dates are telling me.

**Pull request merged**